tests/ict/basic_start_test.go

```
@@ -28,14 +42,538 @@ func TestBasicBtsgStart(t *testing.T) {
+		require.Empty(t, unbondings)
+	})
+}
+
+func testVesting(ctx context.Context, t *testing.T, chain *cosmos.CosmosChain, admin ibc.Wallet) {
+	t.Parallel()
+
+	t.Run("Base Account", func(t *testing.T) {
+		res, err := chain.AuthQueryAccount(ctx, admin.FormattedAddress())
+		require.NoError(t, err)
+		require.EqualValues(t, "/cosmos.auth.v1beta1.BaseAccount", res.TypeUrl)
+		chain.AuthPrintAccountInfo(chain, res)
```

golangci-lint (1.62.2)

Error return value of `chain.AuthPrintAccountInfo` is not checked

(errcheck)